utils/APIFeatures.js

```
@@ -2,16 +2,18 @@ class APIFeatures {
   constructor(query, queryString) {
     this.query = query;
     this.queryString = queryString;
+    this.totalDocuments = 0;
   }
 
   filter() {
     const queryObject = { ...this.queryString };
-    const excludedElements = ['page', 'limit', 'sort', 'feilds'];
+    const excludedElements = ['page', 'limit', 'sort', 'fields'];
 
     excludedElements.forEach((el) => delete queryObject[el]);
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)